apps/website-25/src/components/homepage/CourseSection.tsx

```diff
@@ -13,26 +14,43 @@ const CourseCardWithTracking = withClickTracking(CourseCard, {
 
 const CourseSection = () => {
   return (
-    <SlideList
+    <Section
+      className="course-section"
       title="Our courses"
       subtitle="Our courses combine self-paced study with guided discussions, deepening your learning while fostering connections in the field"
       subtitleLevel="p"
-      className="course-section section-body"
-      maxItemsPerSlide={2}
-      minItemWidth={300}
-      featuredSlot={(
+    >
+      <div className="course-section__content flex flex-col lg:flex-row lg:[&>*]:[flex-basis:50%] gap-space-between items-stretch">
```

Comment:
i prefer to apply styles as directly to the element as possible. i think it improves discoverability and maintainability
```suggestion
      <div className="course-section__content flex flex-col lg:flex-row gap-space-between items-stretch">
```

Reply:
P.S. for future ref, there are also tailwind selectors for children: https://tailwindcss.com/docs/hover-focus-and-other-states#child-selectors
but i do agree with their preceding note:
> While it's generally preferable to put utility classes directly on child elements, you can use the * variant in situations where you need to style direct children that you don’t have control over:

Reply:
This doesn't quite work unfortunately because of the extra `<span>` wrapping. I could add the ability to give the span a className, but I think the current `lg:[&>*]:[flex-basis:50%]` is easier to follow than that.

I think the real answer here is refactoring `withClickTracking` so it doesn't have this problem, probably by making it a hook that components use to implement their own tracking rather than something that is applied externally